### DISCUSSION TITLE
Possibility to turn off logging of fallback?

### DISCUSSION THREAD

**Discussion by elHornair** [intlify/vue-i18n#847] [Q&A] [answered]
Hi there, multilang Vue people!👋

The logging of used fallback locales very verbose. This is useful when debugging the fallbacks, but it fills up the console with useless and distracting information, once the fallbacks are properly setup.

Example:
```
[intlify] Not found 'car' key in 'de-CH' locale messages.
[intlify] Fall back to translate 'car' key with 'de' locale.
```

Is it possible to somehow turn this off?

**Comment by kazupon** [comment, marked as answer]
You can disable with using options the below when call the `createI18n`:

- Options API:
  - `silentFallbackWarn`: set to `true`
  - `silentTranslationWarn`: set to `true`
  
If you use the Composition API, disable with using options when calling the `createI18n`:

- `fallbackWarn`: set to `false`
- `fallbackWarn`: set to `false`

See the docs:
See the docs: https://vue-i18n.intlify.dev/guide/essentials/fallback.html#explicit-fallback-with-one-locale

**Reply by elHornair** [reply]
Thanks @kazupon! 🙌
Sorry for my ignorance, I somehow missed that in the docs.
